charts/gardener/gardenlet/charts/runtime/templates/clusterrole-gardenlet.yaml

```diff
@@ -357,6 +357,8 @@ rules:
   - leases
   verbs:
   - create
+  - list
+  - watch
```

Comment:
Don't we have to update https://github.com/gardener/gardener/blob/7a9282d720b1824e5b789f4819f7140f017cf975/pkg/admissioncontroller/webhooks/auth/seed/authorizer.go#L254-L258 now?

Reply:
Hmm, probably should be updated, but how come I didn't see any problems when testing this with the local setup and `make gardener-up` where the garden cluster and the seed are the same cluster?

Reply:
Maybe the Seed authorizer is not activated in the local setup? For the local setup based on kind/minikube and existing Seed cluster, the Seed authorizer has to be activated explicitly `make ACTIVATE_SEEDAUTHORIZER=true local-garden-up`. Maybe it is not enabled by default also for `make gardener-up`?

Generally, I think @rfranzke has point here, the `allowedVerbs` have to be updated to allow `list` and `watch`.

Reply:
I generally run the local setup (with `make kind-up`) with the seed-admission-controller turned on. I also made sure that the authorization webhook is enabled/configured in the kube-apiserver settings. However, I noticed that there is currently a different error (after rebasing to master).
```
{"level":"info","ts":"2022-10-03T11:55:20.495+0300","logger":"webhook.seedauthorizer","msg":"Denying authorization because attributes are missing object name","seedName":"local","attributes":"authorizer.AttributesRecord{User:(*user.DefaultInfo)(0xc0006efd40), Verb:\"list\", Namespace:\"\", APIGroup:\"core.gardener.cloud\", APIVersion:\"v1beta1\", Resource:\"projects\", Subresource:\"\", Name:\"\", ResourceRequest:true, Path:\"\"}"}
```
I'll dig a bit deeper to see what's happening and how my tests passed previously: